### PR TITLE
Santizes racist robot code from the codebase.

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

**Maintainer edit:**

 Floyd did not make the Mexican tourist bots, the name deportista is also the spanish word for sportsman.

The atomic bomb reference is not racist either, it casts no aspersions on the event that occurred and is simply a reference to a historical fact.

This PR is a character assassination based on faulty reasoning and is not good faith, because those from the past cannot offer a defence of their reasoning.

** Original PR body **
Floyd was cringe like usual, while most of the names are bad, deportista really takes the cake. 

## Why It's Good For The Game

Nobody wants another 'meatball' situation happening to the codebase down the line.

## Changelog

:cl:
qol: Mexican tourist bots are slightly less racist.
qol: Japanese tourist bots will no longer request atomic bombs, what the fuck is wrong with you qust?
/:cl:

### CODE REVIEW

**Comment by DAKKA-WAAAGH** [comment]
Damn, the restaurant parody bots are getting the Old Yeller treatment.

**Comment by carshalash** [comment]
> Damn, the restaurant parody bots are getting the Old Yeller treatment.

God I wish...

**Comment by JohnFulpWillard** [comment]
Sorry but this is all I can think of when I see this 

image

**Comment by tralezab** [comment]
I think of Tourist bots like punch out characters. There's a big difference between the original and the wii version, in terms of... acceptability
"Deportista" sounds way more like original punch out. That's some "Pizza Pasta" or "Vodka Drunkinski" tier naming.

**Comment by Wallemations** [comment]
I didn't even know the bar robots had names.
Glad to see that name get shoved out

**Comment by the-orange-cow** [comment]
Google translate says it means 'sportsman'.

**Comment by Profakos** [comment]
There are so many things that should be cleaned up there, like the japanese robots wanting an Atomic Bomb.

**Comment by carshalash** [comment]
> There are so many things that should be cleaned up there, like the japanese robots wanting an Atomic Bomb.

Not anymore they don't.

**Comment by CRITAWAKETS** [comment]
wait the japanese robots want a **WHAT NOW**

i didnt know qust worked for microsoft goddamn

**Comment by BubbleTrroubles** [comment]
This is like when Blizzard replaced the portraits of women with fruit bowls. I approve our game being more family friendly.

**Comment by carshalash** [comment]
> wait the japanese robots want a **WHAT NOW**, i didnt know qust worked for microsoft goddamn
> 
> on a serious note, i think the drink was the "rare" one for japanese tourists, so it might be good to replace the slot rather than axe it

Intentionally left blank, EOB is working on Japanese-inspired drinks, so one will be added when implemented.

**Comment by EOBGames** [comment]
> > wait the japanese robots want a **WHAT NOW**, i didnt know qust worked for microsoft goddamn
> > on a serious note, i think the drink was the "rare" one for japanese tourists, so it might be good to replace the slot rather than axe it
> 
> Intentionally left blank, EOB is working on Japanese-inspired drinks, so one will be added when implemented.

Man really putting some pressure on me to actually get that finished

**Comment by carshalash** [comment]
> Man really putting some pressure on me to actually get that finished

You have only ever let me down once, and that was by letting tich toch exist.

**Comment by Cyprex** [comment]
These are jokes.

File: code/modules/food_and_drinks/restaurant/customers/_customer.dm [hunk 1]
```diff
@@ -258,7 +258,6 @@
 			/datum/reagent/consumable/cafe_latte = 3,
 			/datum/reagent/consumable/coffee = 3,
 			/datum/reagent/consumable/soy_latte = 3,
-			/datum/reagent/consumable/ethanol/atomicbomb = 1,
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/restaurant/customers/_customer.dm:261]: just have 'em order like, the expensive whiskey from the vending machine

**Comment by optimumtact** [comment]
Please use less emotive titles

**Comment by optimumtact** [comment]
Floyd did not make the Mexican tourist bots, and the name deportista is also the spanish word for sportsman.

The atomic bomb reference is not racist either, it casts no aspersions on the event that occurred and is simply a reference to a historical fact.

This PR is a character assassination based on faulty reasoning and is not good faith, because those from the past cannot offer a defence of their reasoning or the context in which changes were made.

Because of your actions here, we feel we have no recourse other than to give you a 7 day ban for not doing even the most basic of research before trying to attack someone who can no longer defend themselves and directly casting incorrect aspersions of a serious nature in an in game changelog.

**Pull request merged**